debugger/erlide_debugger_23/src/dbg_ieval.erl

```diff
@@ -73,14 +74,17 @@ exit_info(Int, AttPid, OrigPid, Reason, ExitInfo) ->
 	{{Mod,Line},Bs,S} ->
 	    dbg_istk:from_external(S),
 	    Le = dbg_istk:stack_level(),
-	    dbg_icmd:tell_attached({exit_at, {Mod, Line}, Reason, Le}),
+	    dbg_icmd:tell_attached({exit_at, {Mod, Line}, Reason, Le, OrigPid, dbg_istk:all_frames(S), Bs}),
```

Comment:
If I understand this correctly the[ OTP debugger gui](https://github.com/erlang/otp/blob/d6f7f61803591dce1a4631edbd4e311a5786d3f0/lib/debugger/src/dbg_wx_trace.erl#L626) holds state, and knows the OrigPid already. Frames and bindings might have some handling via state as well, not sure.
Just a note that we probably need to update OTPs dbg_wx_trace.erl if we attempt to upstream this.